### PR TITLE
Explain you must pass the 'review' stage before you get prod access

### PR DESCRIPTION
**Pull request by benjamin-mortimer**
This is a condition we have not mentioned in the docs. We'll be seeing some integration connections soon so worth being clear about this here.

### CODE REVIEW

File: source/integrating-with-the-dcs.html.md.erb [hunk 1]
```diff
@@ -34,7 +34,10 @@ To build a working DCS client, you'll need to:
 
 ## Getting access to production
 
-You’ll need to successfully complete a full journey in the test environment before you can get access to the production environment.
+Before you can access production, you'll need to:
+
++ successfully complete a full journey in the test environment
```
Comment (mrwilson) [inline source/integrating-with-the-dcs.html.md.erb:39]: Do we refer to it as a 'test environment' or 'integration environment' elsewhere in the docs? We should be consistent either way.

File: source/integrating-with-the-dcs.html.md.erb [hunk 2]
```diff
@@ -34,7 +34,10 @@ To build a working DCS client, you'll need to:
...
++ show your organisation meet the [DCS pilot requirements](https://www.gov.uk/guidance/requirements-for-joining-the-document-checking-service-pilot) - the DCS team will contact you about this separately
```
Comment (mrwilson) [inline source/integrating-with-the-dcs.html.md.erb]:
```suggestion
+ show your organisation meets the [DCS pilot requirements](https://www.gov.uk/guidance/requirements-for-joining-the-document-checking-service-pilot) - the DCS team will contact you about this separately.
```

File: source/integrating-with-the-dcs.html.md.erb (hunk 1 above)
Comment (benjamin-mortimer) [reply source/integrating-with-the-dcs.html.md.erb]: Yeah, good shout - turns out this is the only reference to it as 'test' so have changed it to 'integration'.

File: source/integrating-with-the-dcs.html.md.erb (hunk 2 above)
Comment (benjamin-mortimer) [reply source/integrating-with-the-dcs.html.md.erb]: Have fixed the verb but the missing full stop is GOV.UK style

**Pull request merged**